modules/apps/frontend-editor/frontend-editor-ckeditor-web/build.gradle

```diff
@@ -19,6 +38,7 @@ buildCKEditor {
 	duplicatesStrategy = "exclude"
 
 	eachFile new StripPathSegmentsAction(2)
+	filter removeBOM
```

Comment:
do we need this in alloy editor too?

Reply:
No. The `ckeditor.js` file in Alloy Editor does not have a BOM at the start. I checked it.

Reply:
See https://unpkg.com/alloyeditor@2.14.10/dist/alloy-editor/ckeditor.js